charts/featureform/charts/coodinator/templates/rolebinding.yaml

```diff
@@ -6,6 +6,9 @@ subjects:
   - kind: ServiceAccount
     name: {{ .Release.Name }}-coordinator
     apiGroup: ""
+  - kind: ServiceAccount
+    name: default
```

Comment:
any reason it is named `default`? just curious

Reply:
its the default account